spec/topological_inventory/orchestrator/test_models/kube_client.rb

```diff
@@ -82,6 +84,10 @@ def delete_secret(name, _namespace)
           secrets.delete(name)
         end
 
+        def get_pods(namespace:, label_selector:)
```

Comment:
@lindgrenj6 Could we just throw a quick `_namespace` here to keep rubocop happy.  Seems to be there pattern above so maybe need to think about a followup PR to adjust for the fact that it is not used anymore.

It's ok to ignore the `find_by` warning for now as that is defined as a private method here and is not a call to the  Rails dynamic finder.

Reply:
Done, I'm curious if we can inform rubocop that we don't even have rails here. He isn't quite sure yet haha.

Reply:
We could look into disabling that cop on the file, see https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code.

We can do that in a followup PR.